VogtPayroll2/PayrollManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    class PayrollManager
+    {
+        private List<Employee> _employees = new List<Employee>();
+
+        public void GetNetPayOfAllEmployees(Employee[] employee)
+        {
+            // The average net pay of all employees should also be displayed
+        }
+
+        public decimal GetMinimumAndMaximumNetPay()
+        {
+            // In addition, the program should find the minimum and maximum net pay of all employees as well as sort the employees based on their net pay (ascending order)
```

Comment:
So it sounds like you already have a good idea on what you need to do here -- but are just trying to figure out how to sort.

Take a look at the LINQ framework in c#. That provides an easy way to sort on a particular field.
https://csharp.net-tutorials.com/linq/sorting-data-the-orderby-thenby-methods/